lib/search-bar/index.js

```diff
@@ -165,20 +181,21 @@ export default class SearchBarContainer extends Component {
         <div className='inputContainer'>
           <img src={searchIcon} className='searchIcon' alt='search'/>
           {inAutoCompleteSearch && <div className='autosuggestSpinner'><LoadingSpinner /></div>}
-          <Autosuggest
-            suggestions={suggestions}
-            onSuggestionsUpdateRequested={this.onSuggestionsUpdateRequested}
-            getSuggestionValue={getSuggestionValue}
-            renderSuggestion={renderSuggestion}
-            inputProps={inputProps}
-            focusInputOnSuggestionClick={false}
-            onSuggestionSelected={(event, { suggestion }) => this.handleOnOptionSelected(suggestion) }
-            multiSection={multiSection}
-            renderSectionTitle={renderSectionTile}
-            getSectionSuggestions={getSectionSuggestions}
-            shouldRenderSuggestions={() => true}
-            focusFirstSuggestion
-          />
+          <div onBlur={() => this.onBlur()}>
```

Comment:
Yeah I tried for quite a while to trigger the onBlur on the component itself. I commented on an issue in Autosuggest   
https://github.com/moroshko/react-autosuggest/issues/20